### PR TITLE
test: (e2e) add e2e tests for Input component

### PR DESCRIPTION
**Pull request by AntonOlkhovskyi**
#### Please provide a link to the associated issue.
closes https://github.com/SAP/fundamental-ngx/issues/4931

#### Please provide a brief summary of this pull request.
add e2e tests for input (core) and makes unit tests run on chrome headless instead of saucelabs
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/core/tests/input.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    });
+
+    it('should check RTL', () => {
+        inputPage.checkRtlSwitch();
+    });
+
+    describe('Check visual regression', function() {
+        it('should check examples visual regression', () => {
+            inputPage.saveExampleBaselineScreenshot();
+            expect(inputPage.compareWithBaseline()).toBeLessThan(3);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/input.e2e-spec.ts:135]: please change to 5%

File: karma.conf.js [hunk 2]
```diff
@@ -5,47 +5,46 @@ const { join } = require('path');
...
-              "--remote-debugging-port=9222"
-          ]
-      }
-  },
-    parallelOptions: {
-      executors: 3
-    },
-  };
+    return {
+        basePath: __dirname,
```
Comment (Seamoo13) [inline karma.conf.js:9]: need to tell about additional changes in PR descr.

File: .travis.yml [hunk 3]
```diff
@@ -71,17 +72,13 @@ jobs:
...
       name: Sauce Labs - unit tests platform
       if: type = pull_request
-      env:
-        - BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
-        - BUILD_ID=$TRAVIS_BUILD_ID
-        - JOB_URL=$TRAVIS_JOB_WEB_UR
-        - SAUCE_USERNAME=$SAUCE_USERNAME
-        - SAUCE_ACCESS_KEY=$SAUCE_ACCESS_KEY
-      script: npm run test:platform:saucelabs
+      before_script:
```
Comment (Seamoo13) [inline .travis.yml:75]: please add info to PR descr.

**Review by Seamoo13 — APPROVED** [review]
approved, but please increase VR error % to 5

**Review by nikvalor — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Review by droshev — APPROVED** [review]

**Pull request merged**